uint16[8] aus_thruster_throttle
---
bool b_result
